images/golang-dind/Dockerfile

```diff
@@ -92,3 +58,9 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
     patch \
     && apt-get clean \
     && python -m pip install --upgrade pip setuptools wheel
```

Comment:
These lines and above are already performed in the bazelbuild image build: https://github.com/jetstack/testing/blob/6b2a9745faf5a307113b94aa73580fde720ac8e8/images/bazelbuild/Dockerfile#L66-L92